dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
Hi @arokem, why do you prefer the ICBM152 2009a version and not the 2009c version?
http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin2009

Also now that you are on it can you also fetch the rest of the maps PDs etc?
We can definitely use them for other projects like the Tissue classifiers etc. 

I would recommend fetching the 1x1x1 atlases but also fetching the 0.5x0.5x0.5 
atlases those will be useful for performing high resolution anatomical tracking.

I hope this is not much to ask. It would be great to fetch the IIT atlas too at some point.
I will need to ask permission from Arfanakis for that. But I think it's possible!
Exciting!